tests/integration/common/s2n_test_scenario.py

```diff
@@ -191,6 +210,7 @@ def get_scenarios(host, start_port, s2n_modes=Mode.all(), versions=[None], ciphe
                 version=version,
                 cipher=cipher,
                 curve=curve,
+                cert=cert,
```

Comment:
Sufficient to resolve issue #1673 ? You removed the TODO referring to it in s2n_test_constants.py, please make sure to link the issue so it can be closed as well, unless there's more work (more cert/key combos to test) to be done.

Reply:
Linked.